databases/week2/school.sql

```diff
@@ -0,0 +1,61 @@
+CREATE DATABASE IF NOT EXISTS School;
```

Comment:
Interesting data model which iseasy to follow! 👍 

The intent of the script also seems to be correct, but you cannot run the script as-is on an empty database. Can you tell why?

A small nitpick is that the column `instructors_id` on `courses` can be a bit misleading as it is only a single instructor.